### ISSUE TITLE
bug: css affect to website which has css contain .is-hidden class

### ISSUE THREAD

**Issue by hieupv-2320** [mozilla/contain-facebook#898]
https://github.com/mozilla/contain-facebook/blob/a7a39f46486b8455d1de24bfba39e65ec95aa06c/src/content_script.css#L352ass

![image](https://user-images.githubusercontent.com/70552882/201305406-a90aaba6-d4ea-48f8-a922-f5e69b2e6414.png)

I have access a website which contains .is-hidden class inside. When I turn on facebook container extension, it hides this class. I've seen patch to fix this (in file content_scipt.css). But it doesn't seem to work or maybe your app hasn't deployed yet . Please review it, thank you at all.

**Comment by krystian3w** [comment]
> maybe your app hasn't deployed yet

Yes, commit https://github.com/mozilla/contain-facebook/commit/a7a39f46486b8455d1de24bfba39e65ec95aa06c don't arrived to public signed xpi (made on 5 Nov and merged ~3 days later, but xpi is form 31 Oct).

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/pull/896

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/issues/886

**Comment by robmadole** [comment]
This also is affecting fontawesome.com/search. We're putting a workaround in place to fix.

**Comment by hieupv-2320** [comment]
@krystian3w  thank you! :D

**Comment by abatkin** [comment]
In case anyone comes searching. [Stumptown Coffee Roasters](https://www.stumptowncoffee.com/) checkout page is broken until this is released.

**Comment by TimmiX88** [comment]
This still hasn't been fixed, i still see the content_script.css being loaded into websites and it's styling affecting the website. Especially the .is-hidden class makes quite a few websites unusable. Please fix/rollout asap.

**Comment by TimmiX88** [comment]
This still hasn't rolled out. @maxxcrawford can you please release this asap? Getting complains from clients that websites aren't working in Firefox and we shouldn't need to make changes to our websites if you just roll out the fix that has been commited months ago.

**Comment by JenniferKiesel** [comment]
I also experience this behaviour on the online banking website of "Sparkasse".

**Comment by krystian3w** [comment]
Someone has some issues more important than spoiling hundreds of pages.

**Comment by krystian3w** [comment]
https://github.com/mozilla/contain-facebook/issues/921
#911

**Comment by krystian3w** [comment]
#919

**Comment by BradyThe** [comment]
That certainly explains the issue I had with a particular website.

For now I am using Stylus in combination with this code to dirty-fix this problem:
`.is-hidden {
    display: inherit !important;
}

#fbc-email.is-hidden, #fbc-login.is-hidden {
    display: none;
}`

**Comment by krystian3w** [comment]
Possible remove these dirty hacks if we try update to 2.3.10 or 2.3.11 from July 2023.
